package.json

```diff
@@ -92,10 +92,12 @@
         "@auth0/auth0-react": "1.10.1",
         "@emotion/react": "11.9.3",
         "@emotion/styled": "11.9.3",
+        "@floating-ui/react-dom": "^1.0.0",
```

Comment:
Please remove the ^ on this and the compose-react-refs

Reply:
Updated.